### ISSUE TITLE
Broken Kyma installation on Gardener in AWS

### ISSUE THREAD

**Issue by adamwalach** [kyma-project/kyma#10491]
**Description**

Every execution of `kyma-integration-gardener-aws` fails:
- k8s1.16: https://storage.googleapis.com/kyma-prow-logs/logs/kyma-integration-gardener-aws/1354791504220000256/build-log.txt
- k8s1.20.2: https://storage.googleapis.com/kyma-prow-logs/logs/halamix2_test_of_prowjob_kyma-integration-gardener-aws/1355168868045164544/build-log.txt

From logs it looks like Kyma installation process was successfull, but it can't be reached from the outside.
Even HTTP is broken (it should return 302 to https):
```
curl http://console.grdb43f2w.kyma-prow.shoot.canary.k8s-hana.ondemand.com -v
*   Trying 15.236.184.150...
* TCP_NODELAY set
* Connected to console.grdb43f2w.kyma-prow.shoot.canary.k8s-hana.ondemand.com (15.236.184.150) port 80 (#0)
> GET / HTTP/1.1
> Host: console.grdb43f2w.kyma-prow.shoot.canary.k8s-hana.ondemand.com
> User-Agent: curl/7.64.1
> Accept: */*
>
* Empty reply from server
* Connection #0 to host console.grdb43f2w.kyma-prow.shoot.canary.k8s-hana.ondemand.com left intact
curl: (52) Empty reply from server
* Closing connection 0
```

Load balancer configuration looks good:
```
kubectl -n istio-system get service istio-ingressgateway
NAME                   TYPE           CLUSTER-IP     EXTERNAL-IP                                                              PORT(S)                                                                                                                                                      AGE
istio-ingressgateway   LoadBalancer   100.64.39.12   a6a48c891506f48eeafc62b4948da396-731560795.eu-west-3.elb.amazonaws.com   15012:30117/TCP,15021:32360/TCP,80:32203/TCP,443:30296/TCP,31400:32408/TCP,15443:32234/TCP,15029:32466/TCP,15030:32579/TCP,15031:30843/TCP,15032:30235/TCP   3h2m

```
but it's a bit different than on other cloud providers (dns name instead of IP).
In istio-ingressgateway(pod) logs I can't see any traffic related info, so it looks like it is blocked somewhere else.

One idea is that it may be caused by a broken external IP resolution.

Config details:
- [istio-ingressgateway.yaml.txt](https://github.com/kyma-project/kyma/files/5894482/istio-ingressgateway.yaml.txt)
- [kyma-gateway.yaml.txt](https://github.com/kyma-project/kyma/files/5894483/kyma-gateway.yaml.txt)

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/8196

**Comment by adamwalach** [comment]
Looks like we have a solution - #10535 
Now, few words of explanation.
We noticed that in AWS the load-balancer health-check is set to port 15012:
Apparently this value is automatically taken from `istio-ingresgateway` config - this is the first item from the list. This config was changed when we [upgraded
istio to 1.8](https://github.com/kyma-project/kyma/blame/master/resources/istio/files/istio-operator-cluster.yaml#L85).
